server/api/routes/auth.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const passport = require('passport');
 const dotenv = require('dotenv');
+const User = require('../actions/users');
```

Comment:
should following the naming conventions set by the file name

```suggestion
const users = require('../actions/users');
```